usr/version.h

```diff
@@ -5,8 +5,12 @@
  * iSCSI tools version.
  * This may not be the same value as the kernel versions because
  * some other maintainer could merge a patch without going through us
+ *
+ * Version string should be set by the build system, else we have problems
```

Comment:
You could do this with a small sed version.h.in -> version.h (similar to what Meson might do). Also by adding autoconf to the top level (AC_CONFIG_SUBDIRS) but that is quite possibly more complexity than it's worth for a build system that would then be phased out.

Reply:
Actually, it seems to be working fine.I have it soft-defined in the top-level Makefile, so it can be overridden when building, and the Makefile passes it to all compilations via a define.

And there is no top-level autoconf for open-iscsi. Currently, most of it uses 'make', and just the 'iscsiuio' subdirectory uses autoconf.

Reply:
Right, like I said you'd have to add it and it would probably be a lot more bother than it's worth.